src/Silverback.Integration.Kafka/Messaging/Broker/KafkaConsumer.cs

```diff
@@ -553,9 +556,8 @@ private void CommitOffsetsIfNeeded()
 
         private void CommitOffsets()
         {
-            if (_messagesSinceCommit == 0)
+            if (Endpoint.Configuration.IsAutoCommitEnabled || !Endpoint.Configuration.IsGroupIdSet || _messagesSinceCommit == 0)
                 return;
```

Comment:
I think that the blank line was needed here.

Reply:
Fixed, see: https://github.com/BEagle1984/silverback/pull/172/commits/62e9b8b415772dfaab2ac3f648be3fb709e33cde